src/editor/debug.c

```diff
@@ -282,6 +282,8 @@ static const char *board_var_list[] =
   "playery*",
   "scrolledx*",
   "scrolledy*",
+  "scrolledpx",
+  "scrolledpy",
```

Comment:
These probably don't need to be writable here...

Reply:
Made read-only.